Adyen/services/management/terminal_settings_terminal_level_api.py

```diff
@@ -11,36 +11,37 @@ class TerminalSettingsTerminalLevelApi(AdyenServiceBase):
     def __init__(self, client=None):
         super(TerminalSettingsTerminalLevelApi, self).__init__(client=client)
         self.service = "management"
+        self.baseUrl = "https://management-test.adyen.com/v1"
```

Comment:
Lets make the version of the endpoint a constant as well so the merchant can use older version. Think you also did that for ruby right?

Reply:
Like this https://github.com/Adyen/adyen-ruby-api-library/blob/97759b0ab4bb9410651dd499b8eedaad1bced725/lib/adyen/services/checkout.rb#L13

Reply:
Also the base url you could make a class constant which they can modify (so the merchant can change it for testing purposes for example).